templates/Default/admin/Default/1.6/universe_create_planets.php

```diff
@@ -0,0 +1,23 @@
+Working on Galaxy : <?php echo $Galaxy->getName(); ?> (<?php echo $Galaxy->getGalaxyID(); ?>)
+<br /><br />
+
+<?php echo $Form; ?>
```

Comment:
Can you change this to
```php
$template->assign('CreatePlanetsFormHREF', SmrSession::getNewHREF($container));
...
<form method="POST" action="<?php echo $CreatePlanetsFormHREF; ?>">
```
so we can get the full benefits of html handling in the template and get rid of another use of `create_echo_form`
(naming of the vars are up to you, I just tried to choose one that made sense)